cap/modules/schemas/views.py

```diff
@@ -59,29 +59,34 @@ def get(self, name=None, version=None):
         resolve = request.args.get('resolve', False)
         latest = request.args.get('latest', False)
         config = request.args.get('config', False)
+        all = request.args.get('all', False)
 
         if name:
             try:
-                if version:
-                    schema = Schema.get(name, version)
+                if all:
+                    schema = Schema.get_versions(name, all=True)
+                elif version:
+                    schema = [Schema.get(name, version)]
```

Comment:
we shouldnt change the type of response from the schema object to an array

Better separate logic